Mail/Views/SplitView.swift

```diff
@@ -101,12 +109,33 @@ struct SplitView: View {
                 }
             }
         }
+        .floatingPanel(isPresented: $isShowingUpdateAvailable) {
+            DiscoveryView(item: .updateDiscovery) { /* Empty on purpose */ } completionHandler: { update in
+                guard update else { return }
+                let url: URLConstants = Bundle.main.isRunningInTestFlight ? .testFlight : .appStore
+                openURL(url.url)
+            }
+        }
+        .floatingPanel(isPresented: $isShowingSyncDiscovery, content: {
```

Comment:
Can we do something like this?
```suggestion
        .floatingPanel(isPresented: $isShowingSyncDiscovery) {
```